en/guide/toolchain.md

```diff
@@ -0,0 +1,121 @@
+# Building C++ DroneCore Apps
+
+DroneCore C++ apps are written in standard C++ (c++11) and can be built using your preferred build system, compiler and linker toolchain. The only requirements are:
+- The build system must be able to locate the DroneCore headers and libraries (installed as described [here](../contributing/build.html#install-artifacts)).
+- When using the library **libdronecore.a**, you need to link to a thread library such as *pthread* on a POSIX system (*pthread* is not included in the static library because it is included in *glibc*). 
```

Comment:
Windows uses the .lib right? Does it need to link against a thread library?

Reply:
Yes, on Windows you need a threading library as well.

It's easiest to use the cmake functionality for this, see:
https://github.com/dronecore/DroneCore/blob/d85344800381d85f1e62467630c2d25c4c7ecab6/example/takeoff_land/CMakeLists.txt#L15-L16

and then link to: ${CMAKE_THREAD_LIBS_INIT}

Oh, you have that explained later. I don't remember what the file is called in Windows, could it be:
```
pthreadVC2.lib
pthreadVCE2.lib
pthreadVSE2.lib
```